site/styles/_meta-detail.scss

```diff
@@ -576,6 +584,33 @@
   }
 }
 
+.element-switcher-wrapper {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
```

Comment:
I disagree. I've replaced the margin-left with gap:50px, but this needs flex shrink/grow. I feel like grid is good but it has its place which isnt here. The flex here is pretty clean and its a really good use of it IMO. I tried to do this with grid, but it wasn't happening and it felt like I was trying to use grid for something flex was intended for

Reply:
Well if the element switcher has no content inside it (this can be achieved via Admin), the jet should skip adding it to the DOM.  Otherwise you have a see more/less button floating there doing nothing.  I consider that a bug.  

And if you delete the element switcher component from the DOM, the sponsor image jumps all the way to the left... do we want it to do that or should it stay on the right?

Reply:
In practice it would get too messy hiding the whole element switcher in template as you have to go:
`{{if film.Overview == "" && !subtitles && !captions etc etc ...}}`

I can add margin-left auto to the sponsor div? That way in future when display none is added to element switcher in relish it sticks to the right

Reply:
Ideally the element switcher would be extracted into a separate jet where all of those checks would be less offensive to look at (see `meta_item.jet` or `tagline.jet`).

Reply:
Might be out of scope for this PR, are you happy to approve as is?

Reply:
Is the potential for a see more/less button that does nothing a pre-exisiting bug? If so, we can carve it off to track separately.